src/CommunityToolkit.Maui/Views/Expander/Expander.shared.cs

```diff
@@ -237,6 +212,45 @@ void SetHeaderGestures(in IView header)
 		headerView.GestureRecognizers.Add(tapGestureRecognizer);
 	}
 
+	static async void ForceUpdateLayoutSizeForItemsView(Expander expander)
+	{
+		if (expander.Header is null)
+		{
+			return;
+		}
+
+		Element element = expander;
+		var size = expander.IsExpanded
+				? expander.Measure(double.PositiveInfinity, double.PositiveInfinity, MeasureFlags.IncludeMargins).Request
+				: expander.Header.Measure(double.PositiveInfinity, double.PositiveInfinity);
+		while (element is not null)
+		{
+			if (element.Parent is ListView listView)
+			{
+				(element as Cell)?.ForceUpdateSize();
+			}
+#if IOS || MACCATALYST
+			else if (element is CollectionView collectionView)
+			{
+				var handler = collectionView.Handler as Microsoft.Maui.Controls.Handlers.Items.CollectionViewHandler;
+				var controller = handler?.GetType().BaseType?.BaseType?.BaseType?.BaseType?.BaseType?.GetProperty("Controller", BindingFlags.NonPublic | BindingFlags.Instance | BindingFlags.GetProperty);
```

Comment:
IMHO this doesn't look very solid. I would say the best solution is to work with someone .NET Maui team and try to expose the property that we need or see if there's other way to grab this information

Reply:
It is protected: `protected ItemsViewController<TItemsView> Controller { get; }`
<img width="1348" alt="image" src="https://user-images.githubusercontent.com/33021114/205166018-5c0492c1-b61e-44cb-ab50-a84feeb9b243.png">

Reply:
So let's talk with the team about it. Any chance that you can send the link to the Maui code line that we need to have access to?

Reply:
src/Controls/src/Core/Handlers/Items/ItemsViewHandler.iOS.cs

https://github.com/dotnet/maui/blob/05ad340fd6b5e8d673da21642c4bdc178f890554/src/Controls/src/Core/Handlers/Items/ItemsViewHandler.iOS.cs#L37-L38

Reply:
@pictos / @VladislavAntonyuk Any progress on getting the needed code promoted to `public` or `internal` via the .NET MAUI engineering team?

Reply:
no progress so far